CONTRIBUTING.md

```diff
@@ -188,3 +188,6 @@ Some pointers on how to recognise if we are breaking MVVM:
 * Views that depend on more than 1 viewmodel class.
 
 If it seems not possible to implement something without breaking some of this advice please consult with @danwalmsley.
+
+## Avoid using Grid as much as possible, Use Panel instead 
+If you dont need any row or column splitting for your child controls, just use `Panel` as your default container control instead of `Grid` since it is a moderately memory-intensive control.
```

Comment:
Nit:
```suggestion
## Avoid using Grid as much as possible, use Panel instead 
If you don't need any row or column splitting for your child controls, just use `Panel` as your default container control instead of `Grid` since it is a moderately memory-intensive control.
```